binderhub/templates/page.html

```diff
@@ -2,19 +2,41 @@
 <html>
 <head>
   <title>{% block title %}Binder{% endblock %}</title>
-  {% block head %}
+
+  {% block meta %}
+  {% endblock meta %}
+
+  {# Social media previews #}
+  {% block meta_social %}
+  <meta property="og:title" content="The Binder Project">
```

Comment:
Do we want this on every page, or just the loading page like before?

Reply:
Before we had this also on index, loading and about pages too. On about page before it was partial though. Now with this PR, about page has all social tags, this is the difference. Or should we completely remove it from about page?

Reply:
I think we should double check that it degrades gracefully, but as a general rule I am +1 on it